packages/rainbowkit/src/wallets/walletConnectors/mewWallet/mewWallet.ts

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable sort-keys-fix/sort-keys-fix */
+import { InjectedConnector } from 'wagmi/connectors/injected';
+import { Chain } from '../../../components/RainbowKitProvider/RainbowKitChainContext';
+import { Wallet } from '../../Wallet';
+import { getWalletConnectConnector } from '../../getWalletConnectConnector';
+
+export interface MewWalletOptions {
+  chains: Chain[];
+  shimDisconnect?: boolean;
+}
+
+export const mewWallet = ({
+  chains,
+  shimDisconnect,
+}: MewWalletOptions): Wallet => ({
+  id: 'mewWallet',
+  name: 'MEW wallet',
+  iconUrl: async () => (await import('./mewWallet.svg')).default,
+  iconBackground: '#fff',
+  downloadUrls: {
+    android:
+      'https://play.google.com/store/apps/details?id=com.myetherwallet.mewwallet&referrer=utm_source%3Drainbow',
+    ios: 'https://apps.apple.com/app/apple-store/id1464614025?pt=118781877&mt=8&ct=rainbow',
+  },
+  createConnector: () => {
+    const isMewWalletInjected =
+      typeof window !== 'undefined' &&
+      Boolean(
+        (
+          window.ethereum as typeof window.ethereum &
+            (undefined | { isMEWwallet?: boolean })
+        )?.isMEWwallet
+      );
+
+    if (isMewWalletInjected) {
+      return {
+        connector: new InjectedConnector({
+          chains,
+          options: { shimDisconnect },
+        }),
+      };
+    }
+
+    const connector = getWalletConnectConnector({ chains });
+
+    return {
+      connector,
+      desktop: {
+        getUri: async () => {
+          return 'https://download.mewwallet.com?source=rainbow';
```

Comment:
These `getUri` functions are supposed to return URIs for connecting to WalletConnect, desktop apps, etc. rather than being used for download links. You can reference the other wallets to see how this works. If you only ever use the injected connector, you shouldn't be defining these functions at all, instead defining a `qrcode` download link.

Does MEW support scanning a QR code at all? If not, I'm not sure we actually support prompting for a mobile app download without it 🤔

Reply:
@markdalgleish Hello, I've made some changes here alongside the Enkrypt PR. Can you take a look, please?